ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -1488,3 +1488,32 @@ def test_tensorflow_IdentityN(
         fn_tree="raw_ops.IdentityN",
         input=x,
     )
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float", full=True)
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.Inv"
+    ),
+)
+def test_tensorflow_Inv(
+    dtype_and_x,
+    as_variable,
+    with_out,
```

Comment:
`with_out` should be removed from the fixtures as the function doesn't accept it